packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -829,4 +920,66 @@ function* renderAttributePart(
   }
 }
 
+/**
+ * Returns a debug string suitable for an error message describing a
+ * TemplateResult.
+ */
+function displayTemplateResult(
+  result: TemplateResult | CompiledTemplateResult
+) {
+  if (isCompiledTemplateResult(result)) {
+    return result._$litType$.h.join('${...}');
+  }
+  return result.strings.join('${...}');
+}
+
 const getLast = <T>(a: Array<T>) => a[a.length - 1];
+
+function willExecute(node: Element | Template): boolean {
+  if (!/script/i.test(node.tagName)) {
+    return false;
+  }
+  if (isTemplateNode(node)) {
```

Comment:
Won't this be always false, as `<template>` would be caught in the !script regex above?

Reply:
True, was using it for type narrowing, rewrote our script tag check instead.